dev-java/jetty-util/jetty-util-9.3.5.20151012.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+JAVA_PKG_USE="doc source"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Jetty's Utils"
+
+MY_PN="jetty"
+MY_PV="${PV/2015/v2015}"
+MY_P="${MY_PN}-${MY_PV}"
+
+SLOT="$(get_version_component_range 1-2)"
```

Comment:
Okay, yes, there are exceptions that I hadn't considered but I believe they are valid exceptions. It does not make sense to use java-utils-2 directly and that fact is documented. It provides shared functionality between java-pkg-2 and java-pkg-opt-2 and would never be removed lightly. versionator could easily be removed if the eclasses itself didn't use it any more.